README.md

```diff
@@ -1,11 +1,71 @@
-![](https://docs.traefik.io/assets/img/traefik.logo.png)
+![Traefik Logo](https://docs.traefik.io/assets/img/traefik.logo.png)
 
 [Traefik](https://traefik.io) is a modern HTTP reverse proxy and load balancer that makes deploying microservices easy.
 
 Traefik integrates with your existing infrastructure components ([Docker](https://www.docker.com/), [Swarm mode](https://docs.docker.com/engine/swarm/), [Kubernetes](https://kubernetes.io), [Marathon](https://mesosphere.github.io/marathon/), [Consul](https://www.consul.io/), [Etcd](https://coreos.com/etcd/), [Rancher](https://rancher.com), [Amazon ECS](https://aws.amazon.com/ecs), ...) and configures itself automatically and dynamically.
 
 Pointing Traefik at your orchestrator should be the *only* configuration step you need.
 
+## Traefik v3 - Example usage
+
+Enable `docker` provider and dashboard UI:
+
+```yml
+## traefik.yml
+
+# Docker configuration backend
+providers:
+  docker:
+    defaultRule: "Host(`{{ trimPrefix `/` .Name }}.docker.localhost`)"
+
+# API and dashboard configuration
+api:
+  insecure: true
+```
+
+Start Traefik v3:
+
+```sh
+docker run -d -p 8080:8080 -p 80:80 \
+  -v $PWD/traefik.yml:/etc/traefik/traefik.yml \
+  -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
```suggestion
  -v /var/run/docker.sock:/var/run/docker.sock \
```
I'm not sure to understand why it should be mounted
Is this still recommended with v3 ?

Reply:
I can run the image successfully without that command @kevinpollet is this still recommended?